starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -717,6 +693,44 @@ sub check_dither {
             }
         }
     }
+
+
+
+
+    # For eng obs, don't have OR to specify dither, so stop before doing vs-OR comparisons
+    if ( $self->{obsid} =~ /^\d*$/){
+	return if ($self->{obsid} >= $ER_MIN_OBSID);
+    }
+
+    # Get the OR value of dither and compare if available
+    my $bs_val = $dither->{state};
+    my $or_val;
+    if (defined $self->{DITHER_ON}){
```

Comment:
I know it's legacy, but looking at this code for the first time in forever, it is incredibly confusing that there are OR-derived values in `$self` with all caps and inconsistent names.  Shouldn't those be captured in a substructure like `$self->{OR}->{DITHER_ON}` or some-such?

Reply:
I can open an issue and work at this if you really want to focus on cleaning up these data structures.  I don't think this is within scope on this PR.  The problem with this idea of moving them is that I don't even know immediately which ones are the OR-derived values as they just basically get bulk copied

https://github.com/sot/starcheck/blob/229b1f82f4a0a21124d0d66cff1fc82a6ae4fd81/starcheck/src/starcheck.pl#L466

Moving them into an OR substructure would be easy, but then going through every starcheck routine to redirect the references seems like it could end up large-scope.